### ISSUE TITLE
Accesibility

### ISSUE THREAD

**Issue by y-zaky** [fac-12/FlagFinder#29]
Form tags such as <input> should have <label> tags in order to improve accesibility for screen readers.
E.g.

<label for="lastname">Last Name</label>
<input type="text" id="lastname" />

Check out this link https://www.coolfields.co.uk/2011/04/accessible-forms-should-every-input-have-a-label/ for more info 👍
